### ISSUE TITLE
Add ImageOutput support to Playerview

### ISSUE THREAD

**Issue by pawaom** [androidx/media#1144]
Image is not rendered in Exoplayer Playerview

I want to display image as video

I have referred   [this](https://github.com/androidx/media/blob/b930b40a16c06318e43c81771fa2b1024bdb3f29/libraries/exoplayer/src/test/java/androidx/media3/exoplayer/e2etest/ImagePlaybackTest.java#L88)

I am trying this code

 ```
val playlist: MutableList<MediaItem> = mutableListOf<MediaItem>()
val mediaItem =   MediaItem.Builder().setImageDurationMs(30000).setUri("content://media/external/images/media/1000005827").build()  playlist.add(mediaItem)
player = ExoPlayer.Builder(this).build()
        playerView = findViewById(R.id.player_view)
        player.addMediaItems(playlist)
        playerView.player = player
        player.prepare()
 player.play()
```

this creates a blank video with the duration 30000 but nothing is rendered , I have tried multiple images but nothing is rendered,

**Comment by icbaker** [comment]
I think this is expected, because images are emitted via [`ImageOutput`](https://developer.android.com/reference/androidx/media3/exoplayer/image/ImageOutput) and `PlayerView` does not implement this interface.

---

I will mark this as an enhancement, but we currently have no plans to work on this.

**Comment by tonihei** [comment]
This has been implemented and published in 1.4.0

**Comment by chileanne** [comment]
please can you indicate an example of the fix @tonihei
